external/googleapis/protolists/aiplatform.list

```diff
@@ -103,8 +103,16 @@
 @com_google_googleapis//google/cloud/aiplatform/v1:model_monitoring.proto
 @com_google_googleapis//google/cloud/aiplatform/v1:model_service.proto
 @com_google_googleapis//google/cloud/aiplatform/v1:nas_job.proto
+@com_google_googleapis//google/cloud/aiplatform/v1:network_spec.proto
+@com_google_googleapis//google/cloud/aiplatform/v1:notebook_euc_config.proto
+@com_google_googleapis//google/cloud/aiplatform/v1:notebook_idle_shutdown_config.proto
+@com_google_googleapis//google/cloud/aiplatform/v1:notebook_runtime.proto
+@com_google_googleapis//google/cloud/aiplatform/v1:notebook_runtime_template_ref.proto
+@com_google_googleapis//google/cloud/aiplatform/v1:notebook_service.proto
```

Comment:
This looks like a new service.  Consider creating bugs to update `generator_config.textproto` or just make a note for yourself.

Reply:
Shall do.